Document export error when no cost description is present

**Describe the bug**
Document can not be exported, when a cost with no description is present.

**Is this a regression?**
Yes

**Steps to Reproduce**
1. Create a DMP
2. Specify datasets to be able to edit other steps
3. Go to step 10 (costs)
4. Create a cost but leave the description field empty
5. Export document (any template)
6. Observe error message flash

**Expected behavior**
Document should be exported correctly.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Environment**
 - Browser: Firefox
 - Damap version: 4.0.3-SNAPSHOT

**Additional context**
Stacktrace
```
Caused by: java.lang.NullPointerException: Cannot invoke "String.lines()" because the return value of "java.util.ArrayList.get(int)" is null
	at org.damap.base.conversion.AbstractTemplateExportFunctions.insertTableCells(AbstractTemplateExportFunctions.java:173)
	at org.damap.base.conversion.AbstractTemplateExportScienceEuropeComponents.composeTableCost(AbstractTemplateExportScienceEuropeComponents.java:1300)
	at org.damap.base.conversion.AbstractTemplateExportScienceEuropeComponents.tableContent(AbstractTemplateExportScienceEuropeComponents.java:836)
	at org.damap.base.conversion.ExportHorizonEuropeTemplate.exportTemplate(ExportHorizonEuropeTemplate.java:58)
	at org.damap.base.conversion.ExportHorizonEuropeTemplate_ClientProxy.exportTemplate(Unknown Source)
	at org.damap.base.conversion.ExportTemplateBroker.exportTemplateByType(ExportTemplateBroker.java:71)
	at org.damap.base.conversion.ExportTemplateBroker.exportTemplate(ExportTemplateBroker.java:55)
	at org.damap.base.conversion.ExportTemplateBroker_ClientProxy.exportTemplate(Unknown Source)
	at org.damap.base.rest.DmpDocumentResource.exportTemplate(DmpDocumentResource.java:57)

```

--- a/src/main/java/org/damap/base/conversion/AbstractTemplateExportScienceEuropeComponents.java
+++ b/src/main/java/org/damap/base/conversion/AbstractTemplateExportScienceEuropeComponents.java
@@ -1317,11 +1317,10 @@ public abstract class AbstractTemplateExportScienceEuropeComponents
         }
 
         ArrayList<String> docVar = new ArrayList<>();
-        docVar.add(costList.get(i).getTitle());
-        if (costList.get(i).getType() != null) docVar.add(costList.get(i).getType().toString());
-        else docVar.add("");
-        docVar.add(costList.get(i).getDescription());
-        docVar.add(costList.get(i).getCurrencyCode());
+        docVar.add(Optional.ofNullable(costList.get(i).getTitle()).orElse(""));
+        docVar.add((costList.get(i).getType() != null) ? costList.get(i).getType().toString() : "");
+        docVar.add(Optional.ofNullable(costList.get(i).getDescription()).orElse(""));
+        docVar.add(Optional.ofNullable(costList.get(i).getCurrencyCode()).orElse("€"));
         if (costList.get(i).getValue() != null) {
           docVar.add(
               NumberFormat.getNumberInstance(Locale.GERMAN).format(costList.get(i).getValue()));

